src/ngx_http_vhost_traffic_status_module.h

```diff
@@ -16,7 +16,7 @@
 #include "ngx_http_vhost_traffic_status_string.h"
 #include "ngx_http_vhost_traffic_status_node.h"
 
-#define NGX_HTTP_VTS_MODULE_VERSION  "0.1.19.dev.91bdb14"
+#define NGX_HTTP_VTS_MODULE_VERSION  "0.2.0.dev.b740132"
```

Comment:
I wonder if we should make this the current release. Maintaining it as a hash is a bit circular.

```suggestion
#define NGX_HTTP_VTS_MODULE_VERSION  "v0.2.0"
```

Reply:
Seems like a good plan. Let's document this idea as comment on the `#define`